backend/local/backend_plan_test.go

```diff
@@ -178,22 +180,23 @@ func TestLocal_planDestroy(t *testing.T) {
 		t.Fatalf("plan operation failed")
 	}
 
-	if !p.RefreshCalled {
-		t.Fatal("refresh should be called")
+	if !p.ReadResourceCalled {
+		t.Fatal("ReadResource should be called")
 	}
 
 	if run.PlanEmpty {
 		t.Fatal("plan should not be empty")
 	}
 
 	plan := testReadPlan(t, planPath)
-	for _, m := range plan.Diff.Modules {
-		for _, r := range m.Resources {
-			if !r.Destroy {
-				t.Fatalf("bad: %#v", r)
-			}
-		}
+	if plan == nil {
+		t.Fatalf("plan is nil")
 	}
+	// for _, r := range plan.Changes.Resources {
```

Comment:
Commented out, currently getting back an empty plan. I opted to have the tests "successfully" fail instead of panic.

Reply:
For these situations where something is stubbed out it's helpful to put in a temporary `t.Fatal` call calling that out, so we can be sure not to forget to deal with this when we do the next pass of test fixing.

Reply:
Yes! That's what the lines above it are for, but I'll add a comment because that's completely non-obvious. Good call-out!